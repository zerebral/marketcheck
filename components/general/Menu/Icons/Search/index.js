import SVG from './search.svg'
import styled from 'styled-components'

const Search = styled(SVG)`
  width 15px;
  fill: #ffffff;
`

export default () => <a><Search /></a>
