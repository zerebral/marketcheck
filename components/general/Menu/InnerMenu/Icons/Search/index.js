import SVG from './search.svg'
import styled from 'styled-components'

const A = styled.a`
  display: block;
`
const Search = styled(SVG)`
  fill: #fff;
  width: 15px;
`

export default () => <A><Search /></A>
