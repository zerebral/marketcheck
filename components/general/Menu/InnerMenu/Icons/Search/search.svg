<svg width='14.86' height='14.62' viewBox="0 0 14.86 14.62">
  <path d="M14.15,14.62,9.75,10.14a5.83,5.83,0,1,1,.68-.73l4.43,4.51ZM5.83,1a4.83,4.83,0,1,0,4.83,4.83A4.84,4.84,0,0,0,5.83,1Z" />
</svg>
