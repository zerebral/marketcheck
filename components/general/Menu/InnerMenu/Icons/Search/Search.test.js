/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import Search from './index'

describe('Search Component', () => {
  test('Search Passes the snapshot', () => {
    const component = renderer.create(<Search />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
