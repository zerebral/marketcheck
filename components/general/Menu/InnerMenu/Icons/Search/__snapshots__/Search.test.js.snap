// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Component Search Passes the snapshot 1`] = `
<a
  className="Search__A-r2sj56-0 injnFb"
>
  <svg
    className="Search-r2sj56-1 ixjDcK"
    viewBox="0 0 14.86 14.62"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M14.15 14.62l-4.4-4.48a5.83 5.83 0 1 1 .68-.73l4.43 4.51zM5.83 1a4.83 4.83 0 1 0 4.83 4.83A4.84 4.84 0 0 0 5.83 1z"
    />
  </svg>
</a>
`;
